main.go

```diff
@@ -228,6 +233,18 @@ func _main() error {
 			continue
 		}
 
+		if arg == "-f" || arg == "--file" {
```

Comment:
Totally unrealted but @Posrabi I realized why the importFilesIteration code is more complex than it should be.

I ridiculously kept -s as a single boolean flag rather than have it look for a string argument after it like this flag here.

I'm going to swap -s for this and it should simplify your code a bit. We'll just be able to pass in the mimetype string and won't need to modify `files` or anyhting.